dap-etl/transformation/src/test/scala/org/broadinstitute/monster/dap/cslb/CslbTransformationSpec.scala

```diff
@@ -54,17 +55,38 @@ class CslbTransformationSpec extends AnyFlatSpec {
   }
 
   it should "raise when date is invalid" in {
-    val invalidDateRecord = RawRecord(id = 1, Map("cslb_date" -> Array("2020-142-124")))
+    val invalidDateRecord = RawRecord(
+      id = 1,
+      Map("cslb_date" -> Array("2020-142-124"), "redcap_event_name" -> Array("annual_2020_arm_1"))
+    )
     assertThrows[DateTimeParseException] {
       CslbTransformations.mapCslbData(invalidDateRecord)
     }
   }
 
+  it should "raise when redcap_event_name is not provided" in {
+    val invalidEventRecord = RawRecord(id = 1, Map("cslb_date" -> Array("2020-11-19")))
+    assertThrows[IllegalStateException] {
```

Comment:
Should I better capture the errors here or would these assertThrows I have here suffice?